.github/workflows/publish_to_pypi.yml

```diff
@@ -1,6 +1,8 @@
 name: Publish django-guid to PyPI and TestPyPI 📦
 
-on: push
+on:
```

Comment:
Not sure why this used to be triggered on push.

Reply:
Not sure if this release thing is new, I missed it or it wasn’t documented, but that was the only way I found to catch tags.   
I did that together with this:
```

 if: github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags') && !contains(github.ref, '-rc')
```


This is way nicer though, thanks!